workbench-core/example/app/src/configs/dynamicRouteConfig.ts

```diff
@@ -62,6 +62,28 @@ export const dynamicRoutesMap: DynamicRoutesMap = {
         }
       }
     ]
+  },
+  '/audit/write': {
+    POST: [
+      {
+        action: 'CREATE',
+        subject: {
+          subjectId: '*',
+          subjectType: 'auditEntry'
+        }
+      }
+    ]
+  },
+  '/audit/is-audit-complete': {
+    POST: [
+      {
+        action: 'CREATE',
```

Comment:
I would probably have made this route a GET with a READ permission

Reply:
changed